lib/css/CssParser.js

```diff
@@ -39,7 +39,7 @@ const IMAGE_SET_FUNCTION = /^(-\w+-)?image-set$/i;
 const OPTIONALLY_VENDOR_PREFIXED_KEYFRAMES_AT_RULE = /^@(-\w+-)?keyframes$/;
 const OPTIONALLY_VENDOR_PREFIXED_ANIMATION_PROPERTY =
 	/^(-\w+-)?animation(-name)?$/i;
-const IS_MODULES = /\.module(s)?\.\w+$/i;
+const IS_MODULES = /\.module(s)?\.[\w-]+$/i;
```

Comment:
changed regex to consider filenames like `styles.module.my-css`

Reply:
Let's add a test case for it, also let's change and use `.+` to handle all characters here

Reply:
Changing to `.+` will also match `some.module.abc.abc` which is invalid. In the future, we need to check the extensions array and match based on that

Reply:
Just use `[^.]+`

Reply:
We should avoid parsing extensions - it is slow